### ISSUE TITLE
Buttons do not have an accessible name

### ISSUE THREAD

**Issue by var-bin** [Varbintech/next.varbintechcom#19]
[Report](https://pagespeed.web.dev/analysis/https-next-varbintech-com/gi57zm9jqs?form_factor=mobile)
---
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers. [Learn how to make buttons more accessible](https://dequeuniversity.com/rules/axe/4.6/button-name).

![14-22-35-9iy4y](https://user-images.githubusercontent.com/3194163/227202588-556c275b-9016-48ed-8443-fdca3a99e1ef.jpg)
